pkg/ccl/backupccl/backup_job.go

```diff
@@ -296,8 +297,8 @@ func backup(
 			// Set a unique filename for each partition backup descriptor. The ID
 			// ensures uniqueness, and the kv string appended to the end is for
 			// readability.
-			filename := fmt.Sprintf("%s_%d_%s",
-				backupPartitionDescriptorPrefix, nextPartitionedDescFilenameID, sanitizeLocalityKV(kv))
+			filename := fmt.Sprintf("%s_%d_%s", backupPartitionDescriptorPrefix,
+				nextPartitionedDescFilenameID, backupinfo.SanitizeLocalityKV(kv))
```

Comment:
[future] We use "Sanitize" in some places to mean "redaction" and then here we use it to mean "make a name suitable to be a filename".  We can probably clean up the naming here.